environment_osx-arm64.yml

```
@@ -17,5 +17,12 @@ dependencies:
   - matplotlib
   - pip
   - pip:
+    - matplotlib
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)